src/Abp.Zero/Zero/Localization/Source/AbpZero.xml

```diff
@@ -1,39 +1,40 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Please rollback changes on AbpZero.xml since they are only changing spaces to tabs.
And please do not use tabs for indents (and change other tabs to spaces too). We are using spaces as convention.